variants/NUCLEO_L476RG/PeripheralPins.c

```diff
@@ -283,3 +283,14 @@ const PinMap PinMap_CAN_TD[] = {
     {PB_9,   CAN1, STM_PIN_DATA(STM_MODE_AF_PP, GPIO_NOPULL, GPIO_AF9_CAN1)},
     {NC,    NP,    0}
 };
+
+//*** WAKE UP ***
+
+const PinMap PinMap_WKUP[] = {
+    {PA_0,  NP,  1},  //WKUP1
+    {PC_13, NP,  2},  //WKUP2
+//  {PE_6,  NP,  3},  //WKUP3 - Not connected
```

Comment:
I would keep it anyway and leave it up to the user to select a pin that is configured.